terraform/aws/modules/module_ecs_service/sg_rules.tf

```diff
@@ -37,3 +37,35 @@ resource "aws_security_group_rule" "healthcheck_to_service_from_load_balancer" {
   security_group_id = aws_security_group.service_sg.id
   source_security_group_id = aws_security_group.service_lb_sg.id  
 }
+
+# Healthcheck rules in case of Network LB
+
+# resource "aws_security_group_rule" "healthcheck_from_netwrok_load_balancer_to_service" {
```

Comment:
@LukaszKwasniewski you commented out this section. Should we leave it here?

Reply:
It can be removed, meant to use it, but it turned to be not needed